README.md

```diff
@@ -45,10 +45,7 @@ Setting up Kong for Kubernetes is as simple as:
 
 ```shell
 # Using Helm
-helm repo add kong https://charts.konghq.com
-helm repo update
-
-helm install kong/kong --generate-name --set ingressController.installCRDs=false
+helm install kong --generate-name -n kong --create-namespace --repo https://charts.konghq.com
```

Comment:
Will this `update` the repo cache 🤔 ?

If not then it's better to use the separate steps as before to ensure users get the latest available version

Reply:
Yes.

If we pass the repository path via the `--repo` flag, Helm will save it to a temporary directory and fetch the latest index (similar to `helm repo update`), ensuring that it always has access to the latest version.

Reply:
Alright, good 👍 

While we're at it, I have thought: why don't we advertize `ingress` chart here already? @mheap Are we ready to do it?

`kong` chart is still installing the sidecar deployment which we're trying to deprecated. With `ingress` we'll promote the split deployments installation

How about:

```suggestion
helm install kong --namespace kong --create-namespace --repo https://charts.konghq.com ingress
```

Reply:
Given that https://github.com/Kong/kubernetes-ingress-controller/issues/4671 says that 

> Make `kong/ingress` the only recommended way of installing KIC in the documentation.

I believe this is the change we'd want to make

Reply:
Thank you for pointing this out, I will try deploying it using the ingress helm chart